cloud-deployment.md

```diff
@@ -3,4 +3,11 @@ title: Cloud Deployment
 toc: false
 ---
 
-Coming soon.
\ No newline at end of file
+To deploy an insecure development or test cluster on Google Cloud Engine using [Terraform](https://www.terraform.io/), see the configuration files and instructions in the [gce directory](https://github.com/cockroachdb/cockroach/blob/master/cloud/gce) of our main cockroach GitHub repository. 
```

Comment:
come to think of it, we may as well put both AWS and GCE.
eg: `see the configuration files and instructions in the [gce] or [aws] directories`